doc/content/the-things-stack/migrating/migration-tool/troubleshooting/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Troubleshooting"
+description: "Troubleshooting the Migration tool"
+weight:
+---
+
+<!--more-->
+
+This section provides help for common issues you may have while using the migration tool.
+
+## error:pkg/redis:not_found (entity not found)
+
+This error could occur when the device is not properly registered in all {{% tts %}} components(`Identity Server`, `Join Server`, `Network Server`, and `Application Server`) and trying to export the device using the migration tool. Below is the example error:
+
+```
+Error: error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
+error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
+    device_id=newdev1
+    correlation_id=e99d3b6b49ce47b08ebbe8f44d57a783
+--- error:pkg/redis:not_found (entity not found)
+    correlation_id=33fabc577f084b6c998ab3b0c1382745
```

Comment:
```suggestion
error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
error:pkg/redis:not_found (entity not found)
```

Reply:
I know my suggestion show the full error, but the rest of lines don't really contribute to understanding in any way and influence readability, so there's no much need for them to be there